src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -380,16 +427,15 @@ public void onTestFailure(ITestResult result) {
         log("onException",
             "driver has no ability to catch screenshot or html source - driver may died", false);
       }
-
       String exception = escapeHtml(result.getThrowable().toString() + "\n"
           + ExceptionUtils.getStackTrace(result.getThrowable()));
+      //TODO: Check imageCounter
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)